<?xml version="1.0"?>
<launch>

    <arg name="robot_id" default="robot" />
    <arg name="use_sim" default="false" />
    <arg name="route" default="1"/>

    <group ns="$(arg robot_id)">

        <node pkg="rbares_pallet" type="pallet_manager.py" name="pallet_manager_node" output="screen">
            <param name="use_sim" value="$(arg use_sim)" />
            <param name="route" value = "$(arg route)" />
        </node>

    </group>

</launch>
